protocol/x/clob/keeper/liquidations.go

```diff
@@ -152,6 +155,27 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 			)
 			return err
 		}
+		clobPairId := k.mustGetClobPairForPerpetualId(ctx, perpetualId).Id
+		subticks := liquidationOrder.GetOrderSubticks()
+		isBuy := liquidationOrder.IsBuy()
+		// Send on-chain deleveraging event for each fill.
+		for _, fill := range fills {
+			k.GetIndexerEventManager().AddTxnEvent(
+				ctx,
+				indexerevents.SubtypeDeleveraging,
+				indexerevents.DeleveragingEventVersion,
+				indexer_manager.GetBytes(
+					indexerevents.NewDeleveragingEvent(
+						subaccountId,
+						fill.OffsettingSubaccountId,
+						clobPairId,
+						satypes.BaseQuantums(fill.FillAmount),
+						uint64(subticks),
+						isBuy,
+					),
+				),
+			)
+		}
```

Comment:
Are there any tests we need to update to check that these new events are correctly emitted?